vars/runStackContinuousPipeline.groovy

```diff
@@ -28,6 +28,7 @@ def call() {
                 steps {
                     sh 'mkdir bin'
                     sh "curl -sL https://raw.githubusercontent.com/crossplane/crossplane-cli/${CROSSPLANE_CLI_RELEASE}/bootstrap.sh | env PREFIX=${WORKSPACE} RELEASE=${CROSSPLANE_CLI_RELEASE} bash"
+                    sh "curl -sL -o bin/semver https://raw.githubusercontent.com/fsaintjacques/semver-tool/master/src/semver && chmod +x bin/semver"
```

Comment:
Can we use commit tag instead of `master` so that we can be sure the executable we run is always the same?

Reply:
Good call

Reply:
Done